languages/elixir/exercises/concept/basic-strings/.docs/instructions.md

```diff
@@ -0,0 +1,67 @@
+In this exercise, you are going to help high school sweethearts profess their love on social media by generating an ASCII heart with their initials:
+
+```
+     ******       ******
+   **      **   **      **
+ **         ** **         **
+**            *            **
+**                         **
+**     J. K.  +  M. B.     **
+ **                       **
+   **                   **
+     **               **
+       **           **
+         **       **
+           **   **
+             ***
+              *
+```
+
+### 1. Get the name's first letter
+
+Implement the `HighSchoolSweetheart.first_letter/1` function. It should take a name and return its first letter. It should clean up any unnecessary whitespace from the name.
+
+```elixir
+iex> HighSchoolSweetheart.first_letter("Jane")
+"J"
+```
+
+### 2. Format the first letter as an initial
+
+Implement the `HighSchoolSweetheart.initial/1` function. It should take a name and return its first letter, uppercase, followed by a dot. Make sure to reuse `HighSchoolSweetheart.first_letter/1` that you defined in the previous step.
+
+```elixir
+iex> HighSchoolSweetheart.initial("Robert")
+"R."
+```
+
+### 3. Split the full name into the first name and the last name
+
+Implement the `HighSchoolSweetheart.initials/1` function. It should take a full name, consisting of a first name and a last name separated by a space, and return the initials. Make sure to reuse `HighSchoolSweetheart.initial/1` that you defined in the previous step.
+
+```elixir
+iex> HighSchoolSweetheart.initials("Lance Green")
+"L. G."
+```
+
+### 4. Put the initials inside of the heart
+
+Implement the `HighSchoolSweetheart.pair/2` function. It should take two full names and return the initials. Make sure to reuse `HighSchoolSweetheart.initials/1` that you defined in the previous step.
+
+```elixir
+iex> HighSchoolSweetheart.pair("Barbara Miller", "William Lewis") |> IO.puts()
+     ******       ******
+   **      **   **      **
+ **         ** **         **
+**            *            **
+**                         **
+**     B. M.  +  W. L.     **
+ **                       **
+   **                   **
+     **               **
+       **           **
+         **       **
+           **   **
+             ***
+              *
```

Comment:
I had to use `IO.puts` to properly demonstrate the newlines (otherwise it would say `"     ******       ******\n   **..."`), but I removed the `:ok` that would be actually be printed there, not to confuse beginners. I feel a bit like I'm lying by thing this... is there a better way to demonstrate this output?

Reply:
We have been generally not featuring `iex` in these exercises, even though it is a huge part of a normal elixir workflow.  I think across the board, we don't have a good way of highlighting how to use a language's REPL (if one exisits) so it has been considered mostly out of the scope of v3 as of now.

I would consider just writing the expected output as a comment following the function call.

Reply:
Could the output be defined as a heredoc?

Reply:
I reformatted all code examples not to use `iex`, and ended up presenting this output as a heredoc.

Reply:
Excellent!